proxy.go

```diff
@@ -152,9 +154,20 @@ func NewProxyFromConfig(logger *logrus.Logger, conf ProxyConfig) (p Proxy, err e
 
 	p.TraceClient = trace.DefaultClient
 	if conf.SsfDestinationAddress != "" {
+		stats, err := statsd.NewBuffered(conf.StatsAddress, 4096)
```

Comment:
Oh, this confused me for a bit. This is a reporter for the tracing client that's proxy uses.

Can we package this up in the trace client somehow rather than leaking it out here? (this is a poorly thought out question)

Reply:
I thought about this, but reporting with statsd is a nice emergency hatch: If the client should super fail, we can still report that it's not working with statsd.

Reply:
Also, I think giving the user of the client their choice of statsd client (buffered/unbuffered, other weird combinations?) feels like the better default. That setup happens only once per program, so I'm not too worried about the duplication (:

Reply:
I was referring to it's home in server.go versus in the traceclient. Could instantiating the traceclient do this work and be configured there? (Maybe that's a mix of concerns tho!)

Reply:
Yep, that's why it's an option to the client constructor - we normally don't want that for all the clients, say in testing; but "serious" stuff will want it, and can provide config files and such.